.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
+vendor-plugin
```

Comment:
Remove this line (and remove the `vendor-plugin` folder locally if it exits).

Maybe `composer.lock` should be added in `.gitignore` to make sure that it's not accidentally committed.